proactive-healthcheck/api-health-check.sh

```diff
@@ -0,0 +1,604 @@
+#!/bin/bash
+#set -x
+
+#input parameters
+#username to connect to the ProActive Portals
+username=<CHANGE_ME>
+#password of the user connecting to the ProActive Portals
+proactiveAdminPasswordDecrypted=<CHANGE_ME>
+#server DNS name or public IP
+serverDNSName=try.activeeon.com
+#server Port
+webPort=8443
+#web protocol, http or https
+webProtocol=https
+#name of one of the node sources, needed in a test to get its description
+nsName=local
+#timeout in seconds to check free nodes, job submission, and job status
+timeoutInSeconds=120
```

Comment:
120 seconds for a REST call is too much. Other platforms generally use 30s. I suggest to use 30.

Reply:
@nebilbenmabrouk it is not a normal timeout for a rest call, it is a loop timeout, for example:
- it could be that the proactive-examples is still loading so the script cannot submit basic-examples/native-task
- it could be that the node deployement took time for some reason 
- it could be that the submitted job is still pending for some reason